### PR TITLE
3.0.0b2

### PR DESCRIPTION
**Pull request by azazellochg**
- chimeraX support
- ctf refine protocol & viewer refactored a bit
- bug fix for wrong image size in optics table
- major fix for 3.0 conversions

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -38,7 +38,7 @@ b) Developer's version
...
+RELION sources will be downloaded and compiled automatically with the plugin, but you can also link an existing installation. Default installation path assumed is ``software/em/relion-3.1``, if you want to change it, set *RELION_HOME* in ``scipion.conf`` file to the folder where the RELION is installed. If you need to use CUDA different from the one used during Scipion installation (defined by *CUDA_LIB*), you can add *RELION_CUDA_LIB* variable to the config file. Moreover, if you have to use a MPI for Relion different from Scipion MPI, you can set *RELION_MPI_BIN* and *RELION_MPI_LIB* variables in the config file.
```
Comment (delarosatrevin) [inline README.rst:41]: @azazellochg I think you introduced some changes that force the version to be X.Y.Z, with.  the .Z part mandatory...so we will need relion-3.1.0 as default.

**Comment by azazellochg** [comment]
Good point, yes, it's called 3.1.0

**Comment by delarosatrevin** [comment]
Yes, I had the issue a couple times,  because 3.1 is not longer recognized as a valid version anymore.

**Comment by azazellochg** [comment]
ChimeraX viewers still not working so I cannot merge this.

**Pull request merged**